app/views/layouts/application.html.haml

```diff
@@ -47,6 +47,10 @@
   = succeed '.' do
     = link_to "Find out more about cookies", cookies_path
 
+- content_for :after_header do
```

Comment:
Can a view template contain a `content_for` block actually? Not sure if they need to be in `application.html.haml` and loaded before the `= render template` line.